driver-async/src/main/com/mongodb/async/client/FindIterableImpl.java

```diff
@@ -154,6 +154,12 @@
         return this;
     }
 
+    @Override
+    public FindIterable<TResult> hintString(@Nullable final String hint) {
```

Comment:
I think there could be a test for this in `com.mongodb.internal.async.client.FindIterableSpecification#should build the expected findOperation`

Reply:
Done